stubs/pdo.phpstub

```diff
@@ -12,7 +12,7 @@ class PDOStatement implements Traversable
      *
      * @template T
      * @param class-string<T> $class
-     * @param array $ctorArgs
+     * @param list<mixed> $ctorArgs
```

Comment:
Just curious whether it supports named arguments in form of `['arg3' => 123, 'param2' => 'zzz']`

Reply:
really good point. just tested locally, and thats confirmed named args are supported